tests/tck/features/aggregate/Agg.feature

```diff
@@ -565,6 +565,15 @@ Feature: Basic Aggregate and GroupBy
       | "Carmelo Anthony" | 1.5 | 1     |
       | "Dwyane Wade"     | 1.5 | 1     |
 
+  Scenario: Distinct sum
+    When executing query:
+      """
+      UNWIND [1,2,3] AS d RETURN d | YIELD sum(distinct $-.d) AS sum
```

Comment:
Test the distinct, use the repeated element  [1,2,3,3]

Reply:
Good catch!